include/crypto/ml_kem.h

```diff
@@ -171,8 +182,7 @@ typedef struct ossl_ml_kem_key_st {
     struct ossl_ml_kem_scalar_st *s;        /* Private key secret vector */
     uint8_t *z;                             /* Private key FO failure secret */
     uint8_t *d;                             /* Private key seed */
-    int prefer_seed;                        /* Given seed and key use seed? */
```

Comment:
I dont think your descriptions here should completely be removed - maybe comment on the added flags